File: Moviles/1º Evaluacion/Ejercicio1/app/src/main/java/com/example/ejercicio1/MainActivity.java
package com.example.ejercicio1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button boton = (Button) findViewById(R.id.boton);

        boton.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
            EditText nombre = (EditText) findViewById(R.id.nombre);
            String nombreTxt = nombre.getText().toString();

            TextView texto = (TextView) findViewById(R.id.texto);
            String tesxtoTxt = texto.getText().toString();

            String txt = tesxtoTxt + "\nBuenos dias " + nombreTxt;
            texto.setText(txt);
        });

    }
}

File: Moviles/1º Evaluacion/Ejercicio1B/app/src/main/java/com/example/ejercicio1b/MainActivity.kt
package com.example.ejercicio1b

import android.R
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val boton = findViewById<Button>(R.id.boton)

        boton.setOnClickListener {
            val nombre = findViewById<EditText>(R.id.nombre)
            val nombreTxt = nombre.text.toString()
            val texto = findViewById<TextView>(R.id.texto)
            val textoTxt = texto.text.toString()
            val txt = "$textoTxt\nBuenos días $nombreTxt"
            texto.text = txt
        }
    }
}